iconic/UserProfile/serializers.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from iconic.UserProfile.models import Wallet
+
+
+# This is custom wallet serializer
+class CustomWalletSerializer(serializers.ModelSerializer):
+    user_id = serializers.OneToOneField(required=True)

```

Comment:
There is no OneToOneField in serializers, 
try user_id = serializers.PrimaryKeyRelatedField(read_only=True)